app/file-upload/server/lib/FileUpload.js

```diff
@@ -23,6 +23,7 @@ import { roomTypes } from '../../../utils/server/lib/roomTypes';
 import { hasPermission } from '../../../authorization/server/functions/hasPermission';
 import { canAccessRoom } from '../../../authorization/server/functions/canAccessRoom';
 import { fileUploadIsValidContentType } from '../../../utils/lib/fileUploadRestrictions';
+import { isValidJWT, generateJsonWebToken } from '../../../utils/server/lib/JWTHelper';
```

Comment:
What do you think of renaming to `generateJWT`, just like the same pattern on the `isValidJWT` ?